docs/modules/ROOT/pages/quarkus-azure-storage-blob.adoc

```diff
@@ -154,6 +154,13 @@ You can go back to the http://portal.azure.com/[Azure portal] and see the contai
 
 image::quarkus-azure-storage-blob-azure-portal2.png[alt=Azure Portal showing the containt of the file]
 
+=== Reactive code pitfall
+
+In case you plan to use `BlobServiceClient` (that provides all other clients for interacting with Azure BlobStore), note that it cannot be used within the reactive code as it blocks the event loop due the the fact that `BlobServiceClient` is a wrapper around the `BlobServiceAsyncClient` and it uses blocking calls such as `blockingGet`.
```

Comment:
Typo `due the the fact` => `due to the fact`?

Reply:
fixed